feeadjuster/test_feeadjuster.py

```diff
@@ -153,37 +162,43 @@ def test_feeadjuster_imbalance(node_factory):
     chan_B = l2.rpc.listpeers(l3.info["id"])["peers"][0]["channels"][0]
     scid_A = chan_A["short_channel_id"]
     scid_B = chan_B["short_channel_id"]
-    l2_scids = [scid_A, scid_B]
+    nodes = [l1, l2, l3]
```

Comment:
[It's the one that's not used](https://github.com/lightningd/plugins/pull/160/files/5b27b0b355fb175c4f3c9461e387463581adbaa5#r531724517)